src/core/zowe/core_for_zowe_sdk/exceptions.py

```diff
@@ -39,9 +39,7 @@ def __init__(self, expected, received, request_output):
             The output from the request
         """
         super().__init__(
-            "The status code from z/OSMF was {} it was expected {}. \n {}".format(
-                received, expected, request_output
-            )
+            "The status code from z/OSMF was {} it was expected {}. \n {}".format(received, expected, request_output)
```

Comment:
_(Unrelated to this PR)_ Thoughts on adjusting this exception message to something like the following?
```suggestion
            "The status code from z/OSMF was: {}\nExpected: {}\nRequest output: {}".format(received, expected, request_output)
```